### DISCUSSION TITLE
<header><footer> inside <main>

### DISCUSSION THREAD

**Discussion by massimoghm** [bootscore/bootscore#689] [General]
Hi, I have read differing opinions regarding the use of <header> and <footer> within <main>. I have noticed that the theme uses this formatting in the various templates (single.php etc). What do you think, could this be a problem for accessibility?

Regards

**Comment by crftwrk** [comment]
You're right, we will check this. Thank you for pointing this out.

**Comment by massimoghm** [comment]
@crftwrk Do you have an idea of the release date for version 6? I'm supposed to start developing a couple of sites and I don't want to have to retrofit both shortly after release. Thanks!

**Reply by crftwrk** [reply]
It's scheduled for the middle of March and includes breaking changes. But there are still some hard things to do and release could be delayed. You can check the current status in the [project board](https://github.com/orgs/bootscore/projects/17). Upon release, we will publish an article in the docs on how to migrate from v5 to v6 step-by-step. Hope that helps.
